pkg/cli/image/manifest/manifest.go

```diff
@@ -214,6 +214,7 @@ type FilterFunc func(*manifestlist.ManifestDescriptor, bool) bool
 // PreferManifestList specifically requests a manifest list first
 var PreferManifestList = distribution.WithManifestMediaTypes([]string{
 	manifestlist.MediaTypeManifestList,
+	imagespecv1.MediaTypeImageIndex,
```

Comment:
AFAICS, `imagespecv1.MediaTypeImageManifest` was added to support OCI manifests in [here](https://github.com/openshift/oc/pull/697/files#diff-88a19aa652d0dad2300f5b2080f14494546c5036b5b357e07ae150e24d67c072R216), what is the reason of adding this one?

Reply:
ah, I missed that sorry. You're right.

Reply:
I should have titled it right :)